integ-test/src/test/java/org/opensearch/sql/multicluster/OpenSearchMultiClustersRestTestCase.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+package org.opensearch.sql.multicluster;
+
+import org.apache.hc.core5.http.HttpHost;
+import org.opensearch.client.Request;
+import org.opensearch.client.RestClient;
+import org.opensearch.common.util.io.IOUtils;
+import org.opensearch.test.rest.OpenSearchRestTestCase;
+import org.junit.AfterClass;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.Collections.unmodifiableList;
+
+/**
+ * Superclass for tests that interact with multiple external test clusters using OpenSearch's {@link RestClient}.
+ */
+public abstract class OpenSearchMultiClustersRestTestCase extends OpenSearchRestTestCase {
```

Comment:
if OpenSearchSQLRestTestCase is the base class of all the IT class. why not just add multicluster capability to OpenSearchSQLRestTestCase?

Reply:
makes sense. done